### PR TITLE
I1050 indexing - comments addressed

### PR DESCRIPTION
**Pull request by jellymlg**
fixed a few mistakes

### CODE REVIEW

File: src/main/resources/api/openapi.yaml [hunk 1]
```diff
@@ -281,7 +281,7 @@ components:
           type: boolean
           example: false
         globalIndex:
-          description: The global index of the box
+          description: Global index of the output in the blockchain
```
Comment (aslesarenko) [inline src/main/resources/api/openapi.yaml:284]: Still not clear what this index means. Has it anything to do with block height?

File: src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala [hunk 2]
```diff
@@ -30,6 +30,9 @@ case class BlockchainApiRoute(readersHolder: ActorRef, ergoSettings: ErgoSetting
 
   private val paging: Directive[(Int, Int)] = parameters("offset".as[Int] ? 0, "limit".as[Int] ? 5)
 
+  /**
+    * Total number of boxes/transactions that can be requested at once
+    */
   private val MaxItems = 16384
```
Comment (aslesarenko) [inline src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala:36]: ok, but why this exact number is maximum. What is you motivation for choosing it?

**Review by aslesarenko — CHANGES_REQUESTED** [review]

File: src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala (hunk 2 above)
Comment (jellymlg) [reply src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala:36]: MaxHeaders is set to this number in BlocksApiRoute to "avoid too heavy requests", so I thought it is reasonable to use as a limit

File: src/main/resources/api/openapi.yaml (hunk 1 above)
Comment (jellymlg) [reply src/main/resources/api/openapi.yaml:284]: The indexer runs from block 1 to the current height.
In every block from transaction 1 to the total number of transactions in that block.
In every transaction from output 0 to the total number of outputs in that transaction.
Every output processed increases globalIndex by 1.
So the globalIndex could be thought of as a kind of 'serial number' for boxes. And also the total number of boxes ever created.

Comment (aslesarenko) [reply src/main/resources/api/openapi.yaml:284]: Perfect, this text should be somewhere in the code and referred in descriptions.

File: src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala (hunk 2 above)
Comment (aslesarenko) [reply src/main/scala/org/ergoplatform/http/api/BlockchainApiRoute.scala:36]: Again, this explanation better be in ScalaDoc.

**Review by aslesarenko — APPROVED** [review]

**Pull request merged**